src/ai/backend/manager/config.py

```diff
@@ -333,7 +333,7 @@
         t.Key("type", default="docker"): t.String,
         t.Key("username", default=None): t.Null | t.String,
         t.Key("password", default=None): t.Null | t.String,
-        t.Key("project", default=None): t.Null | tx.StringList | t.List(t.String),
+        t.Key("project", default=""): t.String(allow_blank=True),
```

Comment:
```suggestion
        t.Key("project", default=None): t.Null | tx.StringList(allow_blank=True) | t.List(t.String),
```

As long as I checked `tx.StringList` validator, it can allow blank string with the `allow_blank` option. Please try it!